CHANGES.md

```diff
@@ -2,6 +2,11 @@
 
 ## Unreleased
 
+### New features
+
+* ci: add workflow to update the release branch. by @gustavosr8 in
+  https://github.com/cnpem/epics-in-docker/pull/86/commits
+
```

Comment:
This is only a house-keeping change. Users shouldn't bother about this.

Take this changelog as "what changes users need to know". IMHO, if they want to see any changes that were made, they should `git log` instead.